src/Stratis.Bitcoin.Features.Miner/MinerSettings.cs

```diff
@@ -70,6 +75,11 @@ public MinerSettings(NodeSettings nodeSettings)
                 this.WalletPassword = config.GetOrDefault<string>("walletpassword", null); // No logging!
             }
 
+            uint blockMaxSize = (uint) config.GetOrDefault<int>("blockmaxsize", (int) nodeSettings.Network.Consensus.Options.MaxBlockSerializedSize, this.logger);
```

Comment:
I wonder if there should be a check here to not allow bigger values then what consensus requires?